Add Outage Banner (#1568)
cowprotocol/cowswap: config/announcements/announcements-100.md

> We are currently experiencing network issues on Gnosis Chain, trading experience may be degraded.

Reviewer: ```suggestion
Gnosis Chain is currently experiencing network issues on, trading experience may be degraded on this network.
```
